protocol/handshake/README.md

```
@@ -0,0 +1,103 @@
+
+## Protocol Identifiers
+
+| Property | Value |
+|----------|-------|
+| Protocol Name | `handshake` |
+| Protocol ID | `0` |
+| Mode | Node-to-Node / Node-to-Client |
+
+## State Machine
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)